synthetic/sub-01/ses-01/beh/sub-01_ses-01_task-stroop_beh.tsv

```diff
@@ -1,3 +1,3 @@
-trial_type	response	reaction_time	stim_file
```

Comment:
I reran the shell script I took from the master branch to generate those TSV files.

So the changes in the TSV are most likely due to the fact that we changed the script in #295 but did not re run it before merging.

https://github.com/bids-standard/bids-examples/pull/295#issuecomment-952876596